src/Starcounter.VS.HTMLTemplateItemTemplate/HTMLTemplate.html

```diff
@@ -4,7 +4,10 @@
     see: https://www.polymer-project.org/1.0/docs/api/elements/dom-bind
```

Comment:
Could you update the comment to `<dom-bind>` and the link to Polymer 2.0's dom-bind?